AppBuilder/platform/views/ABViewGrid.js

```diff
@@ -1680,11 +1680,16 @@ class ABViewGridComponent extends ABViewComponent {
 
       // default our columnConfig values to our columnHeaders:
       columnHeaders.forEach((c) => {
-         // localStorage won't save any .template() functions.
+         // we want to overwrite our default settings with anything stored
+         // in local storage
          var origCol = objColumnHeaders.find((h) => h.fieldID == c.fieldID);
-         if (origCol?.template) {
-            c.template = origCol.template;
-         }
+         // to do this we use _.merge() on duplicate keys the second passed
+         // object wins the tie
+         var mergedCol = _.merge(c, origCol);
```

Comment:
So the `c` definition should have precedence OVER the `origCol`.  It is only the `.template` field we want to pull forward since that isn't saved into localStorage.  

By merging this way, `origCol` is now the one with precedence.

Reply:
I thought I read it that with lodash merge the first object in the params is the master, then the next, then the next. But I think I misunderstood...I will swap these.